src/components/StylesFromLiteralsService.js

```diff
@@ -219,47 +228,69 @@ goog.provide('ga_stylesfromliterals_service');
         return this.defaultStyle;
       };
 
+      OlStyleForPropertyValue.prototype.setOlText_ = function(olStyle,
+        labelProperty, labelTemplate, properties) {
+        var text;
+        properties = properties || [];
+        if (labelProperty) {
+          text = properties[labelProperty];
+          if (text !== undefined && text !== null) {
+            text = text.toString();
+          }
+        } else if (labelTemplate) {
+          text = labelTemplate;
+          angular.forEach(properties, function(prop, k) {
+            if (prop !== undefined && prop !== null) {
```

Comment:
Can `prop` be null or undefined ?  

in any case,  `if (prop) ` should be enough

Reply:
yes anything you could possibly find the properties of a GeoJSON. I can't use the toString method on null or undefined values but an empty string or a zero should be valid,

Reply:
ok